src/installer/tests/HostActivation.Tests/MockCoreClrSanity.cs

```diff
@@ -1,67 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using Microsoft.DotNet.Cli.Build;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Runtime.InteropServices;
-using Xunit;
-
-namespace Microsoft.DotNet.CoreSetup.Test.HostActivation
-{
-    public class MockCoreClrSanity : IDisposable
```

Comment:
Yeah, removed because other tests cover the same product functionality. I think these were also intended to check that the mockcoreclr library itself worked when it was first created - but that's not helpful at this point, since so many tests would fail horribly if the mockcoreclr test infrastructure was broken.